### PR TITLE
Trigger postanl job reliable

### LINKED ISSUE #333: Triggering GFS post_anl job
On 05/25/2021, Russ Treadon reported:

> Lin and Weng,
> 
> The atmos_post_anl job is submitted by the atmos_post_manager when logf000.txt is present.   This trigger is not always reliable.  
> 
> The atmos_post_anl job processes output from job atmos_analysis_calc.   This job creates loganl.txt.   The presence of this file indicates that it is safe to run atmos_post_anl.   Job atmos_analysis submits the forecast job.  The forecast job creates logf000.txt.  Depending on queue wait times, it is possible for the forecast job to create logf000.txt before atmos_analysis_calc creates loganl.txt.  
> 
> Logic in exglobal_atmos_pmgr.sh should be updated to trigger the atmos_post_anl job when loganl.txt, not logf000.txt, is present.  As currently written logf000.txt triggers both atmos_post_anl and atmos_post_f000
> 
>     if [ -s ${COMIN}/${RUN}.${cycle}.logf${fhr}.txt -o  -s ${COMIN}/${RUN}.${cycle}.logf${fhr3}.txt ]
>     then
>       if [ $fhr -eq 0 ]
>       then
> ####        ecflow_client --event release_${RUN}_postanl
>         ecflow_client --event release_postanl
>       fi
> ####      ecflow_client --event release_${RUN}_post${fhr}
>       ecflow_client --event release_post${fhr3}
> 
> This issue also exists in operations.

### PR DESCRIPTION
**Pull request by WenMeng-NOAA**
Update exglobal_atmos_pmgr.sh from Lin Gan to trigger postanl job reliable.

### CODE REVIEW

File: scripts/exglobal_atmos_pmgr.sh [hunk 1]
```diff
@@ -50,7 +50,10 @@ do
       if [ $fhr -eq 0 ]
       then 
 ####        ecflow_client --event release_${RUN}_postanl
-        ecflow_client --event release_postanl
+        if [ -s ${COMIN}/${RUN}.${cycle}.loganl.txt ]
+        then
+          ecflow_client --event release_postanl
+        fi
       fi
```
Comment (KateFriedman-NOAA) [inline scripts/exglobal_atmos_pmgr.sh]: Maybe compress this if-block to become:
```
if [ $fhr -eq 0 -a -s ${COMIN}/${RUN}.${cycle}.loganl.txt ]
then
   ecflow_client --event release_postanl
fi
```
Unless there are plans to add additional cases/if-statements to the `if [ $fhr -eq 0 ]` block.

**Review by KateFriedman-NOAA — APPROVED** [review]
Change looks ok, assume it has been tested. Provided suggestion but approve either way.

**Review by lgannoaa — APPROVED** [review]
Looks good to me.

File: scripts/exglobal_atmos_pmgr.sh (hunk 1 above)
Comment (WenMeng-NOAA) [reply scripts/exglobal_atmos_pmgr.sh]: @KateFriedman-NOAA Good suggestion! I have been waiting for @lgannoaa comments and assume the changes were tested by Lin in ecflow.

**Comment by WenMeng-NOAA** [comment]
@lgannoaa Would you like tweaking based on Kate's suggestion?

**Comment by lgannoaa** [comment]
@WenMeng-NOAA ,
@KateFriedman-NOAA suggestion works. It does not impact the result. I have not test that solution yet because I have a meeting now. I recommend commit as is. Because it is tested.

**Comment by WenMeng-NOAA** [comment]
@lgannoaa Sure. I will commit the current changes soon. We might optimize the workflow scripts in the future. Thanks!

**Review by lgannoaa — APPROVED** [review]
Thank you Wen for help us address this issue so fast!

**Pull request merged**